src/tag/Id3Scan.cxx

```diff
@@ -55,6 +57,12 @@
 #define ID3_FRAME_ALBUM_ARTIST "TPE2"
 #endif
 
+#if LIBMPDCLIENT_CHECK_VERSION(2,12,0)
```

Comment:
Same here. This new tag should always be available, even when not building with libmpdclient.